.evergreen/config.yml

```diff
@@ -586,6 +586,7 @@ functions:
     - command: shell.exec
       type: test
       params:
+        timeout_secs: 10
```

Comment:
Why are we setting a timeout here, and why is it 10 seconds?  Also, the timeout you set in the mocharc sets a default timeout per-test of 10s, whereas here the timeout is on the whole evergreen task

Reply:
This was cruft left over from trying to get the 0-length file path test to run correctly